core/controllers/suggestion_test.py

```diff
@@ -981,12 +1059,6 @@ def test_update_suggestion_updates_question_suggestion_content(self):
 
         self.login(self.CURRICULUM_ADMIN_EMAIL)
         csrf_token = self.get_new_csrf_token()
-        with utils.open_file(
```

Comment:
Why are these parts removed?

Reply:
Because these things are passed to `UpdateQuestionSuggestionHandler` and there we are not expecting any  files, before no error is reported because schemas are not there.

https://github.com/oppia/oppia/blob/5471941da48071723b5f7e8684716584c08c6a1d/core/controllers/suggestion.py#L547

Also if we check frontend we are not passing any file:
https://github.com/oppia/oppia/blob/5471941da48071723b5f7e8684716584c08c6a1d/core/templates/pages/contributor-dashboard-page/services/contribution-and-review-backend-api.service.ts#L198